eskip/eskip.go

```diff
@@ -116,6 +263,15 @@ type Filter struct {
 	Args []interface{} `json:"args"`
 }
 
+func (f *Filter) String() string {
```

Comment:
this package has a filterString method of the route, we could extract its functionality and use it inside this method. See: https://github.com/zalando/skipper/blob/master/eskip/string.go#L122. The reason why that function should be used is that does the escaping right, so that the regexp replace would be executed on the string that user expects.

Reply:
fixed as discussed